core/server/common/src/main/java/alluxio/master/journal/JournalInputStream.java

```diff
@@ -16,16 +16,16 @@
 import java.io.IOException;
 
 /**
- * This input stream retrieves {@link JournalEntry} from journal checkpoint files and journal log
- * files.
+ * This input stream retrieves {@link JournalEntry} from journal checkpoints and journal logs.
  */
 public interface JournalInputStream {
```

Comment:
I think this is because our implementation uses one JournalInputStream per file (either checkpoint or completed log). I would prefer not to make further changes to the API (beyond renaming) in this PR.